src/libs/ReportActionsUtils.js

```diff
@@ -30,6 +30,14 @@ Onyx.connect({
     callback: (val) => (isNetworkOffline = lodashGet(val, 'isOffline', false)),
 });
 
+/**
+ * @param {Object} reportAction
+ * @returns {Boolean}
+ */
+function isCreatedActionType(reportAction) {
```

Comment:
This aligns with the other similar functions we already have, e.g. `isDeletedAction`, `isMoneyRequestAction`, etc
```suggestion
function isCreatedAction(reportAction) {
```